src/vcpkg/configure-environment.cpp

```diff
@@ -101,23 +102,13 @@ namespace vcpkg
         auto& fs = paths.get_filesystem();
         auto& download_manager = paths.get_download_manager();
         auto node_path = paths.get_tool_exe(Tools::NODE, stdout_sink);
-        auto node_modules = paths.root / "node_modules";
-        auto ce_path = node_modules / "vcpkg-ce";
-        auto ce_sha_path = node_modules / "ce-sha.txt";
 
 #if defined(VCPKG_CE_SHA)
+        auto base_path =
+            get_platform_cache_home().value_or_exit(VCPKG_LINE_INFO) / "vcpkg" / "vcpkg-artifacts-" VCPKG_BASE_VERSION_AS_STRING;
```

Comment:
Do we need vcpkg/vcpkg-artifacts... or can we shorten it to, e.g., vcpkg/artifacts...
```suggestion
            get_platform_cache_home().value_or_exit(VCPKG_LINE_INFO) / "vcpkg" / "artifacts-" VCPKG_BASE_VERSION_AS_STRING;
```

Reply:
Fixed!